### PR TITLE
Added the code to toonify any image

### PR DESCRIPTION
**Pull request by Tejas1510**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #294 

### Type of change

**Select all that applies**:
* [x] New feature (non-breaking change which adds functionality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [x] I have added my work in respective language folder.
* [x] My changes generate no new warnings.

### CODE REVIEW

File: Python/ToonifyImage/toonify.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+imagePath=input("Please enter the path of image to toonify : ")
+r = requests.post(
+    "https://api.deepai.org/api/toonify",
+    files={
+        'image': open(imagePath, 'rb'),
+    },
+    headers={'api-key': 'your api key here'}
+)
+y=r.json()
+image_url=y['output_url']
```
Comment (tejan-singh) [inline Python/ToonifyImage/toonify.py:12]: error in line 12

**Review by tejan-singh — CHANGES_REQUESTED** [review]

**Comment by Tejas1510** [comment]
Sir it will show error as on line 9 the user has to enter his own API key after that it will run
I have mentioned it in the code

File: Python/ToonifyImage/toonify.py (hunk 1 above)
Comment (tejan-singh) [reply Python/ToonifyImage/toonify.py:12]: You also need to specify this in your readme

File: Python/ToonifyImage/Readme.MD [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+## How to install above Library
+```
+pip install requests
+pip install shutil
+```
+## How to use it :
+1. Download or clone the repository
+2. Install Required Libraries
+3. Run toonify.py
+4. you will be asked to give the path of image you want to toonify
```
Comment (tejan-singh) [inline Python/ToonifyImage/Readme.MD:21]: specify to add API in readme.

**Review by tejan-singh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**